### PR TITLE
Store flow information in redis

### PR DESCRIPTION
**Pull request by apanzerj**
### Description

Make a call to the flows rest api so we can get the ID of each flow the bot has access to. This will allow me to then send messages to flows via the connector.
### Reviewers

@bhouse

### CODE REVIEW

**Comment by apanzerj** [comment]
@bhouse alternatively I can duplicate the pattern used for user_creator for flows:

``` ruby
module Lita
  module Adapters
    class Flowdock < Adapter
      class FlowsCreator
        class << self
          def create_flow(flow)
            Lita.logger.debug("Creating flow: #{flow['parameterized_name']}")
            Lita.redis.set("flows/#{flow['parameterized_name']}", flow['id'])
          end

          def create_users(flows)
            flows.each do |flow|
              create_flow(flow)
            end
          end
        end
      end
    end
  end
end
```

**Comment by apanzerj** [comment]
Either one works for me :-D

**Comment by apanzerj** [comment]
bump @bhouse ...

File: lib/lita/adapters/flowdock/flow_creator.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module Lita
+  module Adapters
+    class Flowdock < Adapter
+      class FlowsCreator
```
Comment (bhouse) [inline lib/lita/adapters/flowdock/flow_creator.rb]: Where is this getting used @apanzerj?

**Comment by apanzerj** [comment]
Just trying something...

**Comment by apanzerj** [comment]
Gone! Take another look?

**Comment by bhouse** [comment]
so the motivation here is to be able to send messages from a handler without having received a message, and you need the flow's id for that?

**Comment by apanzerj** [comment]
Yup

**Comment by bhouse** [comment]
:+1:

**Pull request merged**